### DISCUSSION TITLE
Please help us change or add the code in local.yaml to hide login form when Google SAML is activated

### DISCUSSION THREAD

**Discussion by EduNetEurope** [kimai/kimai#4914] [Questions & Answers]
We use the self-hosted version of kimai and have successfully activated Google SAML.
Unfortunately, even though the login form is deactivated in the settings, it is still shown on the login page.

@kevinpapst `[suggested that local.yaml can overwrite it.](https://github.com/kimai/kimai/discussions/4725)`

We would be very grateful if someone pointed us to the piece of code that needs to be changed and told us how to change it.

**Comment by kevinpapst** [comment]
Duplicates https://github.com/kimai/kimai/discussions/4725
